src/theme/DefaultTheme.js

```diff
@@ -6,7 +6,7 @@ const core = {
   grayLightest: '#f2f2f2',
 
   borderMedium: '#c4c4c4',
-  border: '#dbdbdb',
+  border: '#ebebeb',
```

Comment:
this seems like the sort of thing that airbnb could provide by a theme - and not require any changes in react-dates that some users might consider semver-major.